content/1.2-rc/docs/installation/requirements.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Requirements"
+date: 2024-11-25
+---
+
+## Operators
+The orchestrator runtime/deployment is made of two main parts: `sonataflow operator` and `RHDH operator`  
+SonataFlow operator resource requirements are described [SonataFlow Installation Requirements](https://sonataflow.org/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html#_prepare_a_minikube_instance)  
```

Comment:
pls add a sub-title so it will be rendered as its own section for  `### OpenShift Serverless Logic operator requirements`
### OpenShift Serverless Logic operator requirements

Reply:
fixed